Lib/sqlite3/__main__.py

```diff
@@ -105,8 +112,11 @@ def main(*args):
         Each command will be run using execute() on the cursor.
         Type ".help" for more information; type ".quit" or {eofkey} to quit.
     """).strip()
-    sys.ps1 = "sqlite> "
-    sys.ps2 = "    ... "
+
+    s = get_theme().syntax
```

Comment:
Since you're using more than one theme section in this script, calling one of them `theme` was misleading IMO. A single-letter name works better in the context of f-strings below.